client-mvp-04/src/utils/uiKit/uiKit.js

```diff
@@ -153,120 +143,120 @@ const DevUiKit = () => {
             <RedirectLink
               linkKey={'footer-page'}
               path={'/page'}
-              className={`tooltip-link`}
+              className="tooltip-link"
               content={` More details here.`}
             />
           </div>
         </div>
-        <p className={'dev-comment'}>Tooltip, works on hover</p>
+        <p className="dev-comment">Tooltip, works on hover</p>
       </div>
 
       {/*** INPUTS ***/}
-      <div className={'inputs-container'}>
-        <h1 className={'kit-sec-title'}>*** INPUTS ***</h1>
+      <div className="inputs-container">
+        <h1 className="kit-sec-title">*** INPUTS ***</h1>
 
         <Input placeholder={'Enter your email'} type={'email'} />
```

Comment:
In this case, the attributes of the Input component don't need {} in my opinion. Typically, {} is used when inputting a variable or some JS functionality. {} aren't needed to input strings.

Reply:
I agree, again.

Reply:
Agree.